addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
-const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : () => {};
+const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : (callback) => callback();
```

Comment:
I know nothing so probably a dumb question, but I'm assuming that callback can never be null/undefined?